packages/@dcl/inspector/src/hooks/sdk/useSdkContext.ts

```diff
@@ -23,9 +25,9 @@ export const useSdkContext = () => {
   }, [dispatch])
 
   useEffect(() => {
-    if (!catalog || !canvas || sdk || isLoading || !dataLayer) return
+    if (!catalog || !canvas || sdk || isLoading || !dataLayer || !preferences) return
```

Comment:
do we need the `!dataLayer` if it's not used anymore by `createSdkContext`?

Reply:
nope, i think that we are covered with the prefences. good catch